.github/workflows/notify-ci-status.yml

```diff
@@ -1,12 +1,17 @@
 name: Notify CI status
 
-on: status
+on:
+  check_suite:
+    types: [completed]
+  status:
 
 jobs:
   call-workflow:
     if: >-
```

Comment:
A comment here explaining that we are supporting two different failure events here would be helpful in the future.